src/Api/Auth/Models/Public/AssociationWithPermissionsBaseModel.cs

```diff
@@ -15,4 +15,9 @@ public abstract class AssociationWithPermissionsBaseModel
     /// </summary>
     [Required]
     public bool? ReadOnly { get; set; }
+    /// <summary>
+    /// When true, the hide passwords permission will not allow the user or group to view passwords.
+    /// </summary>
+    [Required]
+    public bool? HidePasswords { get; set; }
```

Comment:
Making this required is a breaking change, which we should avoid. It should be optional so that existing integrations continue to work, with a default value of `false` (again, maintaining current behaviour).